packages/transport/src/constants.ts

```diff
@@ -1,3 +1,12 @@
+export const MESSAGE_HEADER_BYTE = 0x23;
+export const HEADER_SIZE = 1 + 1 + 4 + 2;
+export const BUFFER_SIZE = 63;
+
+export const T1HID_VENDOR = 0x534c;
```

Comment:
Maybe `T1_HID_VENDOR` would be more clear separating `T1` from `HID`?

Reply:
sure, no problem with that b3a8b0afa86b0426a7fdba05b99ef47aa2839b8d